packages/react-charting/src/components/GaugeChart/GaugeChart.base.tsx

```diff
@@ -237,27 +238,56 @@ export class GaugeChartBase extends React.Component<IGaugeChartProps, IGaugeChar
-                    onBlur={this._handleBlur}
-                    onMouseEnter={e => this._handleMouseOver(e, segment.legend)}
-                    onMouseMove={e => this._handleMouseOver(e, segment.legend)}
-                    data-is-focusable={this._legendHighlighted(segment.legend) || this._noLegendHighlighted()}
-                  />
+                  <React.Fragment key={index}>
+                    <path
+                      d={arc.d}
+                      fill={this.props.enableGradient ? "transparent" : segment.color}
+                      fillOpacity={this._legendHighlighted(segment.legend) || this._noLegendHighlighted() ? 1 : 0.1}
+                      strokeWidth={this.state.focusedElement === segment.legend ? ARC_PADDING : 0}
+                      className={this._classNames.segment}
+                      {...getAccessibleDataObject(
+                        {
+                          ariaLabel: getSegmentLabel(segment, this._minValue, this._maxValue, this.props.variant, true),
+                          ...segment.accessibilityData,
+                        },
+                        'img',
+                        true,
+                      )}
+                      onFocus={e => this._handleFocus(e, segment.legend)}
+                      onBlur={this._handleBlur}
+                      onMouseEnter={e => this._handleMouseOver(e, segment.legend)}
+                      onMouseMove={e => this._handleMouseOver(e, segment.legend)}
+                      data-is-focusable={this._legendHighlighted(segment.legend) || this._noLegendHighlighted()}
+                    />
+
+                    {this.props.enableGradient && (<>
+                      <clipPath id={`clip_${index}_${arc.segmentIndex}`}>
+                        <path d={arc.d} />
+                      </clipPath>
+                      <foreignObject
+                        x="-50%"
+                        y="-100%"
+                        width="100%"
+                        height="100%"
+                        clipPath={`url(#clip_${index}_${arc.segmentIndex})`}
+                      >
+                        <div
+                          style={{
+                            width: '100%',
+                            height: '100%',
+                            background: this.props.enableGradient
+                              ? `conic-gradient(
+                                  from ${arc.startAngle}rad at 50% 100%,
+                                  ${segment.gradient![0]},
+                                  ${segment.gradient![1]} ${arc.endAngle - arc.startAngle}rad
+                                )`
+                              : segment.color,
+                            opacity: this._legendHighlighted(segment.legend) || this._noLegendHighlighted() ? 1 : 0.1,
+                          }}
+                        />
+                      </foreignObject>
+                    </>)}
+                  </React.Fragment>
```

Comment:
Because of this fragment addition all test snapshots have changed. Could you run `yarn update-snapshots` to update them

Reply:
Resolved. For some reason `yarn update-snapshots` as specified in package.json throws errors and doesnt update the snapshots. I had to run `npx jest --updateSnapshot`.